front/app/containers/Authentication/useSteps/stepConfig/lightFlow.ts

```diff
@@ -158,9 +158,6 @@ export const lightFlow = (
     'light-flow:email-confirmation': {
       CLOSE: () => setCurrentStep('closed'),
       CHANGE_EMAIL: async () => {
-        await signOut();
-
-        updateState({ email: null });
         setCurrentStep('light-flow:email');
```

Comment:
This sign out behaviour breaks the flow since it refreshes the page (and therefore brings you out of the authentication flow).

If you remove the signOut, I feel the behaviour is what we want actually. If a user tries to log in during the light user flow with an email and doesn't confirm it + closes the modal, they should still be signed in. If they click the action again, it should open back up to the confirm your email. If you click "change your email" but never enter a new email, you should still be logged into the current email account (not signed out as soon as you click the button?). I think the email/account should only ever be changed if you do enter a new email?

By removing this code, the email is reset when you enter the new email + submit. I've tested the flow and I think it works as expected now. If you have a different opinion though, just let me know :)

Reply:
@jamesspeake Can you think of any potential issues with this approach? :)

Reply:
Slightly confused about the logic here, just going to check it out to follow it through.

Reply:
@luucvanderzee Had a quick sync with James and this behaviour should be fine :+1: :)